pkg/feature/templates/servicemesh/authorino/auth-smm.tmpl

```diff
@@ -0,0 +1,10 @@
+apiVersion: maistra.io/v1
+kind: ServiceMeshMember
+metadata:
+  name: default
+  namespace: {{ .AuthorinoConfigs.Namespace }}
```

Comment:
I flatten it to `.Auth.Namespace` - we have it servicemesh spec already. See my overall comment for the review